ENIGMAsystem/SHELL/Bridges/xlib-OpenGL3/attribs.cpp

```diff
@@ -0,0 +1,16 @@
+#include <GL/gl.h>
+#include <GL/glx.h>
+
+namespace enigma {
+
+int context_attribs[] = {
+    GLX_CONTEXT_MAJOR_VERSION_ARB, 3,
+    GLX_CONTEXT_MINOR_VERSION_ARB, 3,
+    GLX_CONTEXT_PROFILE_MASK_ARB, GLX_CONTEXT_CORE_PROFILE_BIT_ARB,
+    #ifdef DEBUG_MODE
+    GLX_CONTEXT_FLAGS_ARB, GLX_CONTEXT_DEBUG_BIT_ARB,
+    #endif
```

Comment:
This debug should go in the 1.1 bridge too.

Reply:
there is no debug context till ogl 3